gamedrake/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "Slim Moe",
+      "id": "686244278304899077"
+    },
+    "service": "Gamedrake",
+    "description": {
```

Comment:
Service name and folder name must be the same, caps letters, spaces, etc. included.